pkg/test/utils.go

```diff
@@ -163,3 +165,14 @@ func DeserializeJSONToMap(t *testing.T, in string) config.GenericMap {
 	require.NoError(t, err)
 	return m
 }
+
+func WaitFromChannel(in chan []config.GenericMap, timeout int64) ([]config.GenericMap, error) {
+	duration := time.Duration(timeout) * time.Millisecond
```

Comment:
Won't it be more idiomatic to change the type of `timeout` from `int64` to `time.Duration`?

Then, in the invocation side, one can argue that it is more apparent what the unit of time is:
```go
test.WaitFromChannel(ingestOutput, 2 * time.Second)
```

Reply:
Good point thanks!
I changed it.